src/net.h

```diff
@@ -1935,7 +1935,6 @@ friend class CNode;
      */
     static constexpr size_t MAX_UNUSED_I2P_SESSIONS_SIZE{10};
 
-    friend struct CConnmanTest;
```

Comment:
(all commits) does your agent lost all commit description intentionally or it's a bug?

```
    Merge bitcoin/bitcoin#27957: net: remove unused `CConnmanTest`
    
    Co-authored-by: fanquake <fanquake@gmail.com>
```


I prefer to have them for easier lookup to history without opening github

Reply:
I think it's a limitation of the fact I do each one via a PR, and then the PR is squash merged in. I think during the squash merge it is lost.